wesnoth-sof.po

```diff
@@ -2794,36 +2782,25 @@ msgstr "Nyní můžeš nabírat jezdce na gryfonech."
 
 #. [message]: speaker=Glildur
 #: data/campaigns/Sceptre_of_Fire/scenarios/5_Hills_of_the_Shorbear_Clan.cfg:482
-#, fuzzy
-#| msgid ""
-#| "Aha! I’ve spent years looking for you dwarves, but now I’ve found you! "
-#| "Prepare to die!"
 msgid ""
 "Aha! I've been looking all over for you dwarves, but now I’ve found you! "
 "Prepare to die!"
 msgstr ""
-"Ha! Hledáním vás trpaslíků jsem strávil roky, ale teď jsem vás našel! "
-"Připravte se zemřít!"
+"Ha! Hledal jsem vás všude, ale teď jsem vás našel! Připravte se na smrt!"
 
 #. [message]: speaker=Glonoin
 #: data/campaigns/Sceptre_of_Fire/scenarios/5_Hills_of_the_Shorbear_Clan.cfg:486
-#, fuzzy
-#| msgid "Who the devil are you?"
 msgid "Who in the earth’s guts are you?"
-msgstr "Kdo k čertu jsi?"
+msgstr "Kdo u zemských řití jsi ty?"
```

Comment:
no já nevím, vidím v tom spíš u střev matky země, něco jako trpasličí chodby, nebo u zemského jádra, ale řitě?

Reply:
A kdo by si ulevoval způsobem: U střev matky země? To už i plantážník u Rychlých šípů je ostřejší. Ano, pořád to není ono, ale střeva matky země či zemské jádro mi svou ostrostí připomínají <a href="https://www.youtube.com/watch?v=bDJG-hKtk3w" rel="nofollow">masivní létákovou kampaň</a>.